plugins/repositories/repositories_inspector.rb

```diff
@@ -90,7 +92,42 @@ def inspect_yum_repositories
       )
     end
 
-    RepositoriesScope.new(repositories)
+    RepositoriesScope.new(repositories, repository_system: "yum")
+  end
+
+  def inspect_apt_repositories
+    content = system.read_file("/etc/apt/sources.list")
+    begin
+      content += "\n" + system.run_command(
+        "bash", "-c", "cat /etc/apt/sources.list.d/*.list", stdout: :capture
+      )
+    rescue Cheetah::ExecutionFailed
```

Comment:
Why do we need this? This can easily hide errors and lead to weird and hard to debug behavior, so if there's a cleaner way to handle it that would be great.